packages/server-wallet/src/protocols/__test__/defund-channel.test.ts

```diff
@@ -14,15 +13,20 @@ import {ErorringMockChainService, MockChainService} from '../../chain-service';
 import {WalletResponse} from '../../wallet/wallet-response';
 import {ChannelDefunder} from '../defund-channel';
 import {AdjudicatorStatusModel} from '../../models/adjudicator-status';
-import {stateSignedBy, stateWithHashSignedBy} from '../../wallet/__test__/fixtures/states';
-import {alice, bob} from '../../wallet/__test__/fixtures/signing-wallets';
+import {stateSignedBy} from '../../wallet/__test__/fixtures/states';
+import {alice} from '../../wallet/__test__/fixtures/signing-wallets';
 import {Funding} from '../../models/funding';
+import {TestChannel} from '../../wallet/__test__/fixtures/test-channel';
 
 const logger = createLogger(defaultTestConfig());
 const timingMetrics = false;
 
 let store: Store;
 
+const FINAL = 10; // this will be A's state to sign
+const testChan = TestChannel.create({aBal: 5, bBal: 5});
```

Comment:
This is outside the scope of this PR but right now we have `TestChannel` and the `channel` fixture which I think do more or less the same thing. We should probably just standardize on one and remove the other.

Reply:
Good point! https://github.com/statechannels/statechannels/issues/3270